setup.cfg

```diff
@@ -61,3 +61,25 @@ ignore =
 
 [devpi:upload]
 formats = sdist.tgz,bdist_wheel
+
+[mypy]
+# check_untyped_defs = True
+# disallow_any_decorated = True
+# disallow_any_explicit = True
+# disallow_any_expr = True
+# disallow_any_generics = True
+# disallow_any_unimported = True
+# disallow_incomplete_defs = True
+# disallow_subclassing_any = True
+# disallow_untyped_calls = True
+# disallow_untyped_decorators = True
+# disallow_untyped_defs = True
```

Comment:
I would delete all the commented out dead configuration -- one can always look at the configuration reference when adding / removing options

Reply:
I think it's useful as a "TODO"/wishful thinking and for quick experimentation. However, if you don't agree, I'll remove.

Reply:
I would delete all the commented out dead configuration